lib/sass/calcite-web/base/_config.scss

```diff
@@ -3,10 +3,10 @@
 // └──────────────┘
 //  ↳ http://esri.github.io/calcite-web/sass/#ui-variables
 //  ↳ sass → _ui-variables.md
-$transition:               150ms linear                !default;
-$box-shadow:               0 0 16px 0 rgba(0,0,0,.05)  !default;
-$drawer-width:             280px                       !default;
-$easing-function:          cubic-bezier(0.215, 0.440, 0.420, 0.880) !default;
+$transition:               150ms linear                   !default;
+$box-shadow:               0 0 16px 0 rgba(0,0,0,.05)     !default;
+$drawer-width:             280px                          !default;
+$easing-function:          cubic-bezier(.215,.44,.42,.88) !default;
```

Comment:
I think it would be easier to set a blanket rule for leading zeros than selecting individual methods for inclusion, unless there’s a clear difference in how those numbers are being used. In this case, they are both numbers representing numerical values along a range (0 to 255 for `rgb`, 0 to 1 for `a` as well as the cubic-bezier)